kernel_config

```diff
@@ -1293,10 +1293,17 @@ CONFIG_BAYCOM_SER_FDX=m
 CONFIG_BAYCOM_SER_HDX=m
 CONFIG_BAYCOM_PAR=m
 CONFIG_YAM=m
-# CONFIG_CAN is not set
 CONFIG_IRDA=m
 
 #
+# CAN drivers
+#
+CONFIG_CAN=y
+CONFIG_CAN_RAW=y
+CONFIG_CAN_BCM=y
+CONFIG_CAN_VCAN=y
```

Comment:
Great!

I agree, the Debian configuration looks good and works for us. As you mention, they probably have good reasons to do it that way.

Regarding `CONFIG_CAN_VCAN` vs `CONFIG_CAN_GW`, my guess is that the target application is more likely to use real CAN hardware instead of a virtualized one. For our application we don't really need VCAN since we use the hardware, so I guess we could leave it disabled until someone else needs it.

I'll update the pull request with the Debian configuration.